lib/active_model_serializers/model.rb

```diff
@@ -30,6 +30,8 @@ def updated_at
     end
 
     def read_attribute_for_serialization(key)
+      return send(key) if respond_to?(key)
```

Comment:
Hm, I'm not totally sure this would work. You'd get into this scenario:

```ruby
class Post < ActiveModelSerializers::Model
  attr_accessor :title
end

post = Post.new('title' => 'my title')
post.read_attribute_for_serialization(:title) # => nil
```